client/my-sites/plugins/plugin-details-CTA/style.scss

```diff
@@ -20,9 +20,6 @@
 	.eligibility-warnings {
 		margin: 0;
 
-		.card {
```

Comment:
My goal here is to have as few custom CSS as possible from eligibility warnings at plugin-details-CTA. Maybe we can remove everything from here and apply it only at client/blocks/eligibility-warnings/style.scss?

Reply:
I'm hopeful we can do this too do we have a design decision about what these screens should be changed to?

Reply:
Looks like we do https://github.com/Automattic/wp-calypso/issues/68250#issuecomment-1261499675